src/components/manage/Toolbar/More.jsx

```diff
@@ -126,30 +126,18 @@ class More extends Component {
               {editAction && <Display pathname={path} />}
             </li>
             <li>
-              {historyAction ? (
```

Comment:
I think we need the history conditional for not showing it in the root, isn't it?

Reply:
@sneridagh The problem is that in Plone 4 `historyAction` is undefined everywhere, but `/history` is a valid path.